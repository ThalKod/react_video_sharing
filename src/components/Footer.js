
import React from "react";
import { Link } from "react-router-dom";

import logo from "../assets/images/logo.svg";
import loveIcon from "../assets/images/like.png";


export default function Footer(){
    return (
        <div className="container-fluid">
          <div className="row">
            <div className="container padding-def">
              <div className="col-lg-1  col-sm-2 col-xs-12 footer-logo">
                <Link className="navbar-brand" to="/"><img src={logo} alt="Circle logo" className="logo"/></Link>
              </div>
              <div className="col-lg-7  col-sm-7 col-xs-12">
                <p><i>Inspired from Circle's <a href="https://www.azyrusthemes.com">Azyrus Themes</a></i></p>
                <div className="delimiter"/>
                <div className="f-copy">
                  <p><i>â’¸ Copyrights 2019 - Made with <img src={loveIcon} alt="love icon"/> By <a href="https://www.thal.tech">Thal Marcelin</a></i></p>
                </div>
              </div>
            </div>
          </div>
        </div>
    )
}
