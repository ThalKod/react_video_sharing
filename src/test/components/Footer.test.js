import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Footer from "components/Footer";

describe("Footer component", () => {
  it("should render the footer correctly", () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
