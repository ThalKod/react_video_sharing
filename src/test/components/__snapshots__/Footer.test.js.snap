// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render the footer correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="container padding-def"
    >
      <div
        className="col-lg-1  col-sm-2 col-xs-12 footer-logo"
      >
        <Link
          className="navbar-brand"
          replace={false}
          to="/"
        >
          <img
            alt="Project name"
            className="logo"
            src="logo.svg"
          />
        </Link>
      </div>
      <div
        className="col-lg-7  col-sm-7 col-xs-12"
      >
        <p>
          <i>
            Inspired from Circle's 
            <a
              href="https://www.azyrusthemes.com"
            >
              Azyrus Themes
            </a>
          </i>
        </p>
        <div
          className="delimiter"
        />
        <div
          className="f-copy"
        >
          <p>
            <i>
              â’¸ Copyrights 2019 - By 
              <a
                href="https://www.thal.tech"
              >
                Thal Marcelin
              </a>
            </i>
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
